client/src/components/ResourceHub/AddResourceModal.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { ReactElement, FC, Dispatch } from 'react';
+import { Modal, Button, Form, Input, Checkbox } from 'semantic-ui-react';
+
+import { createResource, isError } from '../../utils/apiWrapper';
+
+import '../../css/AddResourceModal.css';
+
+const teams = [
+  'General',
+  'Editing',
+  'Factchecking',
+  'Illustration',
+  'Photography',
+  'Onboarding',
+  'Visuals',
+  'Writing',
+];
+
+interface IProps {
+  onAdd: Dispatch<void>;
+}
+
+const AddResourceModal: FC<IProps> = ({ onAdd }): ReactElement => {
+  const [open, setOpen] = React.useState<boolean>(false);
```

Comment:
I know it's a style thing but for project consistency just use 

`useState()` and not `React.useState()`